tictactoe-starter-code/script.js

```diff
@@ -0,0 +1,21 @@
+var turn = "a"
+
+
+$('#reset').click(function(){
+  $('#reset').removeClass( "x O")
+})
+
+$('td').click(function(){
```

Comment:
When you target td like this, you're getting back an array of every td element - so you're adding this event listener to every single td on the page, hence why they all get filed with x when any of them is clicked.